src/triton_object_recognition/launch/yolov3.launch.py

```diff
@@ -0,0 +1,44 @@
+import os
+import io
+from ament_index_python.packages import get_package_share_directory
+import launch
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+import pathlib
+import yaml
+from launch_ros.utilities import normalize_parameters
```

Comment:
you can remove any import statements not being used just to get rid of any clutter